python/tank/authentication/core_defaults_manager.py

```diff
@@ -78,11 +77,15 @@ def get_user_credentials(self):
                   User or None in case no credentials could be established.
         """
         if not self._mask_script_user:
-            from . import shotgun
             data = shotgun.get_associated_sg_config_data()
             if data.get("api_script") and data.get("api_key"):
                 return {
                     "api_script": data["api_script"],
                     "api_key": data["api_key"]
                 }
         return super(CoreDefaultsManager, self).get_user_credentials()
+
+
+# For backwards compatibility.
```

Comment:
this feels backwards to me - why not put it in `util`? Maybe double check how we have done in it other places (i think it's a recurring pattern) and follow suit?